code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -390,6 +390,11 @@
 	if(prob(40))
 		M.heal_organ_damage(5 * REM, 0)
 
+/datum/reagent/blood/changeling_blood
+	name = "Blood"
+	id = CHANGELINGBLOOD
+	custom_metabolism = 0 //does not leave your body
```

Comment:
See but real blood DOES metabolize at a normal rate, so you could more easily test blood by injecting it into someone and seeing if it goes down.